src/Microsoft.DotNet.Interactive.Formatting/HtmlFormatter.cs

```diff
@@ -43,13 +42,21 @@ void Initialize()
 
     internal static void FormatAndStyleAsPlainText(
         object value,
-        FormatContext context)
+        FormatContext context,
+        string mimeType = PlainTextFormatter.MimeType)
```

Comment:
It is a bit confusing to see the `PlainText` suffix in the name of the function when the mime type is also specified in this parameter. Could we do something to disambiguate?